grunt.js

```diff
@@ -348,25 +353,17 @@ module.exports = function(grunt) {
     'zip:dist'
   ]);
 
-  // Remove this and add to the test task once mmcc's coverall changes are merged
-  grunt.registerTask('newtest', ['build', 'karma:chrome']);
-
-  // Default task.
-  grunt.registerTask('default', ['build', 'test-local']);
+  // Default task - build and test
+  grunt.registerTask('default', ['test']);
 
-  // Development watch task. Doing the minimum required.
-  grunt.registerTask('dev', ['jshint', 'sass', 'browserify:build', 'karma:chrome']);
+  grunt.registerTask('test', ['build', 'karma:defaults']);
 
-  // Skin development watch task.
-  grunt.registerTask('skin-dev', ['connect:dev', 'watch:skin']);
+  // Run while developing
+  grunt.registerTask('dev', ['build', 'connect:dev', 'concurrent:watchSandbox']);
```

Comment:
Yeah, it works here. It has to be part of a chain of tasks or it just shuts down, in case we want another task that has keepalive: true.